captcha/widgets.py

```diff
@@ -72,15 +72,16 @@ def build_attrs(self, base_attrs, extra_attrs=None):
 class ReCaptchaV3(ReCaptchaBase):
     template_name = "captcha/widget_v3.html"
 
-    def __init__(self, api_params=None, *args, **kwargs):
+    def __init__(self, api_params=None, action="form", *args, **kwargs):
```

Comment:
@Andrew-Chen-Wang if the user does not customize the `action` parameter, it should default to `'form'` which is the previously hard-coded value. This piece _should_ be backwards compatible, then?